packages/kbn-eslint-plugin-eslint/package.json

```diff
@@ -7,7 +7,12 @@
     "eslint": "^5.6.0",
     "babel-eslint": "^9.0.0"
   },
+  "devDependencies": {
+    "@types/eslint": "^4.16.6"
```

Comment:
Doesn't look like we're using these

Reply:
that's actually for better support in IDE https://github.com/elastic/kibana/pull/33697/files#diff-30e414c8b475a24e057ef011e384085dR65
remove?

Reply:
Eh, maybe I'm find keeping it, but only if you plan to getting back in here.